files/default/munge-init

```diff
@@ -38,8 +38,8 @@ DAEMON_EXEC="$sbindir/munged"
 #CONFIG=#_NOT_SUPPORTED_#
 PIDFILE="$localstatedir/run/munge/munged.pid"
 #NICE=
-USER="munge"
-GROUP="munge"
+USER=<%= node['cluster']['munge']['user'] %>
+GROUP=<%= node['cluster']['munge']['group'] %>
```

Comment:
should this be moved to the templates dir now that it's using node attributes?

Reply:
Yes, it should. In `recipes/munge_install.rb` I assumed it to be a template and locally I have it like that. Maybe I forgot to push it